docs/source/frameworks/pytorch.rst

```diff
@@ -21,7 +21,7 @@ For common PyTorch models with single input:
 
 .. code-block:: python
     :caption: `train.py`
-
+    
```

Comment:
this file has some spaces in blank lines, we should remove them

Reply:
got it.